config/default.yml

```
@@ -775,6 +775,12 @@ Rails/ReadWriteAttribute:
   Include:
     - app/models/**/*.rb
 
+Rails/RedundantActiveRecordAllMethod:
+  Description: Detect redundant `all` used as a receiver for Active Record query methods.
```

https://github.com/rubocop/rails-style-guide/pull/340 has been merged. Can you add the Style Guide URL and squash your commits into one?

```suggestion
  Description: Detect redundant `all` used as a receiver for Active Record query methods.
  StyleGuide: 'https://rails.rubystyle.guide/#redundant-all'
```

---

I've addressed it and squashed the commits!